apps/studio/src/features/mail/templates/__tests__/templates.test.ts

```diff
@@ -56,3 +57,179 @@ describe("invitationTemplate", () => {
+    )
+    expect(template.body).toContain(
+      "Your site(s) will continue to be accessible to visitors, and all your work remains intact.",
+    )
+  })
+
+  it("should include instructions for regaining access", () => {
+    // Arrange
+    const template = accountDeactivationTemplate(mockData)
+
+    // Assert
+    expect(template.body).toContain(
+      "To regain access to your site(s), please follow the instructions below",
+    )
+  })
+
+  it("should include site name and admin emails in instructions when admins exist", () => {
+    // Arrange
+    const template = accountDeactivationTemplate({
+      recipientEmail: "test@example.com",
+      sitesAndAdmins: [
+        {
+          siteName: "Test Site 1",
+          adminEmails: ["admin1@example.com", "admin2@example.com"],
+        },
+      ],
+    })
+
+    // Assert
+    const expectedSiteInstructions = `<p><b>Test Site 1</b></p>
+          <p>To regain access, please contact one of your site's administrators:</p>
+          <ul><li>admin1@example.com</li><li>admin2@example.com</li></ul>
+        `
+    expect(template.body).toContain(expectedSiteInstructions)
+  })
+
+  it("should show support email message when no admins exist", () => {
+    // Arrange
+    const template = accountDeactivationTemplate({
+      recipientEmail: "test@example.com",
+      sitesAndAdmins: [
+        {
+          siteName: "Test Site 1",
+          adminEmails: [],
+        },
+      ],
+    })
+
+    // Assert
+    const expectedSiteInstructions = `<p><b>Test Site 1</b></p>
+        <p>There are no administrators for this site. To be added back, please send an email to <a href="${ISOMER_SUPPORT_LINK}">${ISOMER_SUPPORT_EMAIL}</a> with your line manager in CC for approval.</p>
+      `
+    expect(template.body).toContain(expectedSiteInstructions)
+  })
+
+  it("should handle multiple sites with admins", () => {
+    // Arrange
+    const template = accountDeactivationTemplate({
+      recipientEmail: "test@example.com",
+      sitesAndAdmins: [
+        {
+          siteName: "Site A",
+          adminEmails: ["admin1@example.com"],
+        },
+        {
+          siteName: "Site B",
+          adminEmails: ["admin2@example.com", "admin3@example.com"],
+        },
+      ],
+    })
+
+    // Assert
+    const expectedSiteInstructionForSiteA = `<p><b>Site A</b></p>
+          <p>To regain access, please contact one of your site's administrators:</p>
+          <ul><li>admin1@example.com</li></ul>
+        `
+    expect(template.body).toContain(expectedSiteInstructionForSiteA)
+
+    const expectedSiteInstructionForSiteB = `<p><b>Site B</b></p>
+          <p>To regain access, please contact one of your site's administrators:</p>
+          <ul><li>admin2@example.com</li><li>admin3@example.com</li></ul>
+        `
+    expect(template.body).toContain(expectedSiteInstructionForSiteB)
+  })
+
+  it("should handle multiple sites with mixed admin scenarios", () => {
```

Comment:
[test] what happens if you're the admin :sadge: (or if you're the only admin)

Reply:
from reading later on, we'll send out the mail still but say that they should contatc isomser admin; notsure if i saw a test case and i lazy reread lol